index.js

```diff
@@ -75,6 +75,12 @@ var SplunkStreamEvent = function (config) {
         this.defaultMetadata.sourcetype = config.splunk.sourcetype
         delete config.splunk.sourcetype;
     }
+    // If evnetFormatter callback mentioned in the splunk object.
+    this.overrideEventFormatter = null;
+    if (config.splunk.eventFormatter) {
```

Comment:
If `config.splunk.eventFormatter` is supposed to be an object, then this will return true for `{}`.